app/dashboard/tests/test_dashboard_utils.py

```diff
@@ -297,3 +297,16 @@ def test_release_bounty_to_public_is_successful():
         assert bounty.bounty_reserved_for_user is None
         assert bounty.reserved_for_user_from is None
         assert bounty.reserved_for_user_expiration is None
+
+    def test_getBlock_is_sucess():
+        assert getBlock__('latest') is not None
+
+    def test_getBlock_is_BlockNotFound():
```

Comment:
What are these tests actually testing? Seems they only test the connection to the web3 provider, which is largely out of our control or merely tests whether the test environment has the proper API keys set up

Reply:
@danlipert do you think we don't need these tests?  the web3.py lib has good coverage, if necessary I remove these tests.

Reply:
Yeah I think we can remove these